package.json

```diff
@@ -6,7 +6,8 @@
   "license": "BSD",
   "dependencies": {
     "async": "~0.2.10",
-    "bootstrap": "3.3.5"
+    "bootstrap": "3.3.5",
+    "bootstrap-slider": "^6.0.9"
```

Comment:
You mean we should fix the version to use? ie ``6.0.9``?
What about
```
"jquery": "^1.9.1",
"eslint-config-openlayers": "^2.0.0"
```
a few lines below?